lib/src/widgets/movie_body_homepage.dart

```diff
@@ -0,0 +1,102 @@
+import 'package:animate_do/animate_do.dart';
+import 'package:flutter/material.dart';
+import '../blocs/i_movies_bloc.dart';
+import '../models/movie.dart';
+import '../ui/movie_detail.dart';
+import '../utils/movie_dimensions.dart';
+
+class MovieBodyHomePage extends StatefulWidget {
+  final IMoviesBloc moviesBloc;
```

Comment:
Instead of passing all the bloc you can use only the data inside, because you already call the method that add movies into stream on HomePage

Reply:
Another approach that you can do is change widget name, think that here you are only creating a GridView to show movies

Reply:
That's a nice tip. Fixed.